src/eraftpb.rs

```diff
@@ -1,4 +1,4 @@
-// This file is generated by rust-protobuf 2.2.4. Do not edit
+// This file is generated by rust-protobuf 2.2.2. Do not edit
```

Comment:
Can it work with protobuf <2.2? For example, will code generated by new compiler use a new API missing in old version?

Reply:
I haven't seen a build error or test error from this. I think we could even allow "~2.0-~2.2"

Reply:
PTAL